tests/scripts/thread-cert/Cert_7_1_08_BorderRouterAsFED.py

```diff
@@ -0,0 +1,219 @@
+        self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+
+        self.nodes[ROUTER].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[ROUTER].get_state(), 'router')
+
+        self.nodes[FED].start()
+        self.simulator.go(3)
+        self.assertEqual(self.nodes[FED].get_state(), 'child')
+
+        self.collect_rlocs()
+        self.collect_rloc16s()
+        self.collect_leader_aloc(LEADER)
+
+        self.nodes[FED].add_prefix(PREFIX_2001, 'paros')
+        self.nodes[FED].add_prefix(PREFIX_2002, 'paro')
+        self.nodes[FED].register_netdata()
+        self.simulator.go(10)
+
+        self.nodes[FED].stop()
+        self.simulator.go(250)
+
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        LEADER_RLOC = pv.vars['LEADER_RLOC']
+        LEADER_ALOC = pv.vars['LEADER_ALOC']
+        ROUTER = pv.vars['ROUTER']
+        ROUTER_RLOC = pv.vars['ROUTER_RLOC']
+        ROUTER_RLOC16 = pv.vars['ROUTER_RLOC16']
+        FED = pv.vars['FED']
+        FED_RLOC = pv.vars['FED_RLOC']
+        FED_RLOC16 = pv.vars['FED_RLOC16']
+
+        # Step 1: Ensure topology is formed correctly
+        pv.verify_attached('ROUTER', 'LEADER')
+        pv.verify_attached('FED', 'ROUTER', 'MTD')
+
+        # Step 2: FED automatically sends a CoAP Server Data Notification
+        #         message with the server’s information (Prefix, Border Router)
+        #         to the Leader.
+        pkts.filter_wpan_src64(FED).\
+            filter_ipv6_dst(LEADER_ALOC).\
+            filter_coap_request(SVR_DATA_URI).\
+            filter(lambda p: {
+                              Ipv6Addr(PREFIX_2001[:-3]),
+                              Ipv6Addr(PREFIX_2002[:-3])
+                             } == set(p.thread_nwd.tlv.prefix) and\
+                   p.thread_nwd.tlv.border_router_16 == [FED_RLOC16, FED_RLOC16]
+                   ).\
+            must_next()
+
+        # Step 3: Leader transmits a 2.04 Changed CoAP response to the DUT.
+        #         And transmits multicast MLE Data Response with the new information
+        #         collected, adding also 6LoWPAN ID TLV for the prefix set on FED.
+        with pkts.save_index():
+            pkts.filter_wpan_src64(LEADER).\
+                filter_ipv6_dst(FED_RLOC).\
+                filter_coap_ack(SVR_DATA_URI).\
+                must_next()
+        with pkts.save_index():
+            pkts.filter_wpan_src64(LEADER).\
+                filter_LLANMA().\
+                filter_mle_cmd(MLE_DATA_RESPONSE).\
+                filter(lambda p: {
+                                  Ipv6Addr(PREFIX_2001[:-3]),
+                                  Ipv6Addr(PREFIX_2002[:-3])
+                                 } == set(p.thread_nwd.tlv.prefix) and\
+                                {
+                                 NWD_6LOWPAN_ID_TLV
+                                 } <= set(p.thread_nwd.tlv.type) and\
```

Comment:
```suggestion
                                NWD_6LOWPAN_ID_TLV in p.thread_nwd.tlv.type and\
```

Reply:
@FiveDimensions , are you planning to address this?

Reply:
updated, thanks